### PR TITLE
Unit Tests for /sawtooth-core/families/block_info/

### PR DESCRIPTION
**Pull request by sandeeplandt**
The unit tests increase code coverage for block_info module
code coverage increased as per nose2-3 report

Signed-off-by: sandeeplandt <sandeepx.hs@intel.com>

### CODE REVIEW

**Comment by rberg2** [comment]
Hello, I have added you to the whitelist what was preventing this PR from building and restarted the build. Thanks!

File: families/block_info/tests/test_tp_block_info_config.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        """ Tests the merge of all toml config files """
+        l = []
+        block_config_default_1 = load_default_block_info_config()
+        block_config_default_2 = load_default_block_info_config()
+        block_config_default_3 = load_default_block_info_config()
+        l.append(block_config_default_1)
+        l.append(block_config_default_2)
+        l.append(block_config_default_3)
+        mc = merge_block_info_config(l)
+        self.assertEqual(mc.connect, "tcp://localhost:4004")
```
[the remark this answers is not in the document]
Comment (sandeeplandt) [reply families/block_info/tests/test_tp_block_info_config.py]: Hi Dan, Thank you for reviewing the code. I have modified the code based on your comments and pushed the code. I think changes are as per comments. Please check and please let me know your comments for the same

File: families/block_info/tests/test_tp_block_info_config.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    def test_to_toml(self):
+        block_config_default = load_default_block_info_config()
+        self.assertEqual(block_config_default.to_toml_string(),
+                        ['connect = "tcp://localhost:4004"'])
```
[the remark this answers is not in the document]
Comment (sandeeplandt) [reply families/block_info/tests/test_tp_block_info_config.py]: Hi Dan, Thank you for reviewing the code. I have modified the code based on your comments and pushed the code. I think changes are as per comments. Please check and please let me know your comments for the same

File: families/block_info/tests/test_tp_block_info_config.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sandeeplandt) [reply families/block_info/tests/test_tp_block_info_config.py]: Hi Shawn, Thank you for the message. I have added test description to test_tp_block_info_config. Please check and please let me know your comments for the same.

**Comment by vaporos** [comment]
Closing due to DCO issues. Please fix DCO and merge conflicts and resubmit if this is still desired.

**Pull request closed without merging**